Dockerfile

```diff
@@ -13,6 +13,16 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends nmap net-tools cython3 libudev-dev sudo libglib2.0-dev bluetooth libbluetooth-dev && \
     apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+# Add Telldus repository
+RUN echo "deb http://download.telldus.com/debian/ stable main" >> /etc/apt/sources.list.d/telldus.list
```

Comment:
Can you combine all these update commands and merge with the previous apt-get line. For Docker layers to not get out of control it is important to use as little as possible run commands.